ambry-router/src/main/java/com.github.ambry.router/DeleteManager.java

```diff
@@ -100,10 +100,13 @@ void submitDeleteBlobOperation(String blobIdString, FutureResult<Void> futureRes
     try {
       BlobId blobId = RouterUtils.getBlobIdFromString(blobIdString, clusterMap);
       DeleteOperation deleteOperation =
-          new DeleteOperation(routerConfig, responseHandler, blobId, futureResult, callback, time);
+          new DeleteOperation(routerConfig, routerMetrics, responseHandler, blobId, futureResult, callback, time);
       deleteOperations.add(deleteOperation);
     } catch (RouterException e) {
       routerMetrics.operationDequeuingRate.mark();
+      routerMetrics.operationAbortCount.inc();
```

Comment:
It is from your patch:)

I think this metric records all the operations that have not gone through the full process (aborted half way, e.g., before receiving required responses).